### PR TITLE
Rename cartpole with cart

### PR DESCRIPTION
**Pull request by ahcorde**
Related with this PR in `gazebo_ros2_control` https://github.com/ros-controls/gazebo_ros2_control/pull/252

### CODE REVIEW

File: gz_ros2_control_demos/launch/tricycle_drive_example.launch.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def generate_launch_description():
         executable='create',
         output='screen',
         arguments=['-string', doc.toxml(),
-                   '-name', 'cartpole',
+                   '-name', 'cart',
```
Comment (christophfroehlich) [inline gz_ros2_control_demos/launch/tricycle_drive_example.launch.py:60]:
```suggestion
                   '-name', 'tricycle',
```

**Review by christophfroehlich — CHANGES_REQUESTED** [review]
The changes here are missing (still referring to a cart-pole controller): https://github.com/ros-controls/gazebo_ros2_control/pull/252/files#diff-3161f66a61b9ed68303607f1dbdce1c7bdcc69c42f8ea1c2051a07a579ebcf5dR210-R217

**Review by christophfroehlich — CHANGES_REQUESTED** [review]
Sorry, but this is still not a valid ros2_control yaml (there is no `type` or `write_op_modes` parameter). Instead of the two code-blocks in L212,220, you could just use
```
.. literalinclude:: ../gz_ros2_control_demos/config/cart_controller_position.yaml
   :language: yaml
```

**Comment by christophfroehlich** [comment]
Btw the "Check Docs" CI job fails because of another issue with control.ros.org

**Comment by ahcorde** [comment]
@christophfroehlich  not really sure about how to solve this issue

```bash
 /home/runner/work/gz_ros2_control/gz_ros2_control/doc/acknowledgements/acknowledgements.rst:70: CRITICAL: Problems with "raw" directive path:
InputError: [Errno 2] No such file or directory: 'doc/acknowledgements/reviewers_stats.html'.
```

**Review by christophfroehlich — APPROVED** [review]

**Comment by christophfroehlich** [comment]
> @christophfroehlich not really sure about how to solve this issue
> 
> ```shell
>  /home/runner/work/gz_ros2_control/gz_ros2_control/doc/acknowledgements/acknowledgements.rst:70: CRITICAL: Problems with "raw" directive path:
> InputError: [Errno 2] No such file or directory: 'doc/acknowledgements/reviewers_stats.html'.
> ```

As I said, a bug on control.ros.org -> I fixed it in the meantime.

**Comment by ahcorde** [comment]
thank you @christophfroehlich for pushing gazebo_ros2_control and gz_ros2_control

**Comment by ahcorde** [comment]
https://github.com/Mergifyio backport humble iron

**Pull request merged**